File: app/src/main/java/com/example/rickandmorty/presentation/fragments/locations/details/CharacterViewHolder.java
package com.example.rickandmorty.presentation.fragments.locations.details;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.example.rickandmorty.R;

public class CharacterViewHolder extends RecyclerView.ViewHolder {

	TextView tvNameCharacter, tvSpeciesCharacter, tvGenderCharacter, tvStatusCharacter;
	ImageView ivCharacter;

	public CharacterViewHolder(@NonNull View itemView) {
		super(itemView);
		ivCharacter = itemView.findViewById(R.id.image_character);
		tvNameCharacter = itemView.findViewById(R.id.name);
		tvSpeciesCharacter = itemView.findViewById(R.id.species);
		tvGenderCharacter = itemView.findViewById(R.id.gender);
		tvStatusCharacter = itemView.findViewById(R.id.status);
	}

}

File: app/src/main/java/com/example/rickandmorty/presentation/fragments/episodes/details/CharacterViewHolder.kt
package com.example.rickandmorty.presentation.fragments.episodes.details

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.example.rickandmorty.R

class CharacterViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

	var tvNameCharacter: TextView
	var tvSpeciesCharacter: TextView
	var tvGenderCharacter: TextView
	var tvStatusCharacter: TextView
	var ivCharacter: ImageView

	init {
		ivCharacter = itemView.findViewById(R.id.image_character)
		tvNameCharacter = itemView.findViewById(R.id.name)
		tvSpeciesCharacter = itemView.findViewById(R.id.species)
		tvGenderCharacter = itemView.findViewById(R.id.gender)
		tvStatusCharacter = itemView.findViewById(R.id.status)
	}
}